### PR TITLE
Add Dev protocol brand section

### PR DESCRIPTION
**Pull request by mkubdev**
## Related Issue 

N/A

Closes: # 

### Describe the changes you've made

Added the Dev Protocol Brand section.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 cap1 | cap2

### CODE REVIEW

**Review by KukretiShubham — APPROVED** [review]
LGTM 🚀 
We would need to create cards like effect later.

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :) Update this PR with the suggestions :)

**Comment by shuklaritvik06** [comment]
Hey @mkubdev  can you update the ss please  :)

File: index.html [hunk 1]
```diff
@@ -357,19 +364,65 @@ <h3 class="tracking-widest text-yellow-500 text-sm font-semibold font-medium tit
...
+                </p>
+              </div>
+            </a>
+
+            <a href="https://community.devprotocol.xyz/" class="h-full">
+              <div class="mx-auto flex flex-col justify-evenly items-center h-full content-center text-center p-8 shadow cursor-pointer rounded-md focus:outline-none focus:shadow-outline transform bg-white transition hover:shadow-lg hover:scale-105 duration-300 ease-in-out">
+                <img class="object-center h-full object-none"
+                src="./assets/dev_protocol_forum.png" alt="Dev Protocol Forum">
+                <p class="text-gray-800 mt-8 mb-2 text-sm px-8 sm:px-4 h-full">
+                  Dev Protocol forum is an open conversation for everyone who can collaborate or propose an ideas to the Dev Protocol team or help other community members.
```
[the remark this answers is not in the document]
Comment (shuklaritvik06) [reply index.html:419]: > ```suggestion
>                   Dev Protocol forum is an open conversation for everyone who can collaborate or propose ideas to the Dev Protocol team or help other community members.
> ```

Hmm, Good Suggestion David!

**Pull request merged**